README.md

```diff
@@ -29,6 +29,7 @@
 | [cosign](./images/cosign) | `cgr.dev/chainguard/cosign` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cosign.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cosign:latest) |
 | [crane](./images/crane) | `cgr.dev/chainguard/crane` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/crane.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/crane:latest) |
 | [curl](./images/curl) | `cgr.dev/chainguard/curl` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/curl.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/curl:latest) |
+| [debug](./images/debug) | `cgr.dev/chainguard/debug` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/debug.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/debug:latest) |
```

Comment:
We need a better name for this than "debug" since everyone has a debug image.

This is for a specific use case right? Should we name it `<something>-debug`?